classes/mail/Mailer.inc.php

```diff
@@ -225,4 +230,62 @@ protected function shouldSendMessage($message, $data = [])
         $site = $request->getSite();
         return $this->events->until(new MessageSendingSite($site, $message, $data)) !== false;
     }
+
+    /**
+     * @return string[] mailable class names
+     */
+    public static function getMailables(): array
+    {
+        $mailables = static::getMailablesFromCache();
+        HookRegistry::call('Mailer::Mailables', [&$mailables]);
+
+        return $mailables;
+    }
```

Comment:
Would we be able to call this with the Laravel facade? ie - `Mail::getMailables()` or do we need to use our mailer like:

```php
use PKP\mail\Mailer;

Mailer::getMailables();
```

Fine either way, just curious.

Reply:
Yes, should be possible to call through the Facade. The direct ancestor is `MailManager` but the latter implements a magic method (`MailManager::__call`), which make it possible to use methods of the default mailer

Reply:
(`MailManager` uses our `Mailer` rather than Laravel's)